notsobot/notsobot.py

```diff
@@ -1623,12 +1718,16 @@ async def rotate(self, ctx, degrees: int = 90, urls: ImageFinder = None):
 
             def rotate_img(b, degrees):
                 with Image.open(b).convert("RGBA") as img:
-                    img = img.rotate(int(degrees))
+                    image = img.rotate(int(degrees))
+                    img.close()
                 with BytesIO() as final:
-                    img.save(final, "png")
+                    image.save(final, "png")
                     file_size = final.tell()
                     final.seek(0)
-                    file = discord.File(final, filename="rotate.png")
+                    filename = self.random_filename(True, "png")
+                    file = discord.File(final, filename=filename)
+                    image.close()
+                    final.close()
```

Comment:
This is unnecessary since `final` was created by the context manager which closes it automatically.